webpack.config.js

```diff
@@ -47,21 +53,86 @@ var config = {
       })
     ],
   module: {
-    rules: [
-      { test: /\.ts$/, use: "ts-loader" },
-      {
-        test: /\.ts$/,
-        enforce: "pre",
-        loader: "tslint-loader",
-        options: {
-          typeCheck: false,
-          configFile: false,
-          failOnHint: true
-        }
-      }
+    unknownContextRegExp: /$^/,
+    unknownContextCritical: false,
+    exprContextRegExp: /$^/,
+    exprContextCritical: false,
+    wrappedContextCritical: true,
+    rules: [{
+      test: /\.(jsx|js)$/,
+      use: [{
+        loader: "babel-loader"
```

Comment:
The problem is described here. https://github.com/joeeames/WebpackFundamentalsCourse/issues/3 Couldn't found other work around